### DISCUSSION TITLE
How to Recover a deleted repo?

### DISCUSSION THREAD

**Discussion by Arwa8881** [community/community#129417] [Repositories] [answered]
### Select Topic Area

Question

### Body

Can i recover my deleted repo

**Comment by davevad93** [comment]
Hi @Arwa8881 ,

Yes, a deleted repository can be recovered within 90 days, follow these steps:

1. Go to `Settings`
2. Scroll down the taskbar and click on `Repositories`
![1](https://github.com/community/community/assets/94308234/2d971293-1b7e-4976-9ecd-513f56a196e1)

3. Click on `Deleted repositories` 
![0](https://github.com/community/community/assets/94308234/5fb202db-5451-4f64-b2e7-a69ec47a3921)

There you will find a list of all the repos you have deleted in the last 90 days, click on `Restore` to recover it.

**Comment by akibhaider** [comment]
You can checkout official github documentation on restoring deleted repo. This can be categorized mainly on two types depending on your type of repository. 
One is: [Restoring a deleted repository that was owned by an organization](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#restoring-a-deleted-repository-that-was-owned-by-an-organization)
The other one is: [Restoring a deleted repository that was owned by a personal account](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#restoring-a-deleted-repository-that-was-owned-by-a-personal-account)
You will get detailed explanations. Hope it helps!

**Comment by Ahmed8881** [comment, marked as answer]
Yes you can recover your deleted repo 


1-Access Recently Deleted Repositories: Go to your profile, click on "Your repositories," and scroll to the bottom to find recently deleted repositories.

2-Restore the Repository: If your deleted repository appears in the list, click on it and select the option to restore.

3-Confirm and Done: Confirm the restoration process, and GitHub will restore your repository if it's within the recovery window.

**Reply by Ctmax-ui** [reply]
Thx bro it was helpfull

**Reply by ivalpib** [reply]
there is no deleted repo

**Reply by yugesherukula** [reply]
yes there is no deleted repositories

**Comment by jimshalo10** [comment]
There is an important point here in the repository restore information

_If you want to restore a repository that was part of a fork network that is not currently empty, you can contact us through the [GitHub Support portal](https://support.github.com/).

Important

You can only contact GitHub Support to restore a_ **repository if you are on a paid GitHub plan.** _For more information about the different plans, see [GitHub’s plans](https://docs.github.com/en/get-started/learning-about-github/githubs-plans).

It can take up to an hour after a repository is deleted before that repository is available_

This means indepandant developers, who use Epic Engine Developers Fork CANNOT get the Personal Forks/Repositories restored

This is lucky that my  four years of personal Repsitories are backed up on my PC and you should do the same!.

My security warning is if you receive an email from ANYONE asking you to relink your Github account to your Microsoft account or any other account **DELETE THE EMAIL its NOT AUTHENTIC. Thats how my Repository got deleted!!.**
